apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/VolatileResourceManager.cs

```diff
@@ -202,10 +202,10 @@ public async Task<int> ExecuteMutationsAsync(List<Mutation> mutations, Cancellat
             return await transaction.ExecuteMutationsAsync(mutations, cancellationToken, timeoutSeconds).ConfigureAwait(false);
         }
 
-        public async Task<ReliableStreamReader> ExecuteReadOrQueryAsync(ReadOrQueryRequest request, CancellationToken cancellationToken, int timeoutSeconds)
+        public async Task<ReliableStreamReader> ExecuteReadOrQueryAsync(ReadOrQueryRequest request, CancellationToken cancellationToken)
```

Comment:
I cannot make them internal because they are from ISpannerTransaction, but I've changed them to explicitly implement the interface on the last commit.